Model/DataObject/ClassDefinition/Data/ObjectBridge.php

```diff
@@ -16,7 +18,7 @@
 /** @noinspection ClassOverridesFieldOfSuperClassInspection
  * We need to overwrite public properties because pimcore uses them for storing data
  */
-class ObjectBridge extends ClassDefinition\Data\ObjectsMetadata
+class ObjectBridge extends ClassDefinition\Data\ObjectsMetadata implements \DataDictionaryBundle\Interfaces\DataDictionary
```

Comment:
No, you don't to do this. Because then the objectbridge is now depending on the datadictonary